apps/credential-showcase-api-server/src/controllers/__tests__/CredentialSchemaController.integration.test.ts

```diff
@@ -44,9 +44,9 @@ describe('CredentialSchemaController Integration Tests', () => {
       .send({
         name: 'Test Schema',
         version: '1.0',
-        identifierType: 'DID',
+        identifierType: IdentifierType.Did,
         identifier: 'did:test:456',
-        source: 'CREATED',
+        source: Source.Created,
```

Comment:
```suggestion
        source: Source.CREATED,
```


export enum Source {
  IMPORTED = 'IMPORTED',
  CREATED = 'CREATED',
}

Reply:
no